files/build_templates/sonic_debian_extension.j2

```diff
@@ -388,6 +388,10 @@ sudo cp files/dhcp/90-dhcp6-systcl.conf.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMP
 sudo cp $IMAGE_CONFIGS/interfaces/init_interfaces $FILESYSTEM_ROOT/etc/network/interfaces
 sudo mkdir -p $FILESYSTEM_ROOT/etc/network/interfaces.d
 
+# Disable default systemd udev rules for interfaces to prevent
+# systemd from overriding interface's MAC address
+sudo ln -sf /dev/null $FILESYSTEM_ROOT_ETC/systemd/network/99-default.link
```

Comment:
what does the default systemd udev rule doing right now?

where can I find the default systemd udev rule?

Reply:
```
[Match]
OriginalName=*

[Link]
NamePolicy=keep kernel database onboard slot path
AlternativeNamesPolicy=database onboard slot path
MACAddressPolicy=persistent
```

Assigning a generated persistent MAC address on every interface name matching '*' pattern.

Reply:
why this default rule is causing the issue? do you have a full explanation of the problem?